parity/cli/usage.rs

```diff
@@ -404,22 +419,27 @@ macro_rules! usage {
 								)*
 							];
 
-							if subc_usages.is_empty() {
-								help.push_str(&format!("parity [options] {}\n", underscore_to_hyphen!(&stringify!($subc)[4..])));
-							} else {
-								help.push_str(&format!("parity [options] {} {}\n", underscore_to_hyphen!(&stringify!($subc)[4..]), subc_usages.join(" ")));
-							}
+							help.push_str(&fill(
+								format!(
+									"parity [options] {} {}\n",
+									underscore_to_hyphen!(&stringify!($subc)[4..]),
+									subc_usages.join(" ")
+								).as_ref(), term_width)
+							);
 						}
 					}
 				)*
 
+				help.push_str("\n");
+
 				// Arguments and flags
 				$(
-					help.push_str("\n");
 					help.push_str($group_name); help.push_str(":\n");
 
 					$(
-						help.push_str(&format!("\t{}\n\t\t{}\n", $flag_usage, $flag_help));
+						help.push_str(&format!("    {}\n{}\n", $flag_usage,
+							indent(fill($flag_help, term_width-8).as_ref(),"        ")
```

Comment:
Could you extract this into a variable (const INDENT_PREFIX: &str = "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;";). And then you can replace the `8` with `INDENT_PREFIX.len()`.